README.md

```diff
@@ -31,3 +31,53 @@ Check out the [Docs](https://karpenter.sh/docs/) to learn more.
 - 11/19/2021 [Karpenter @ Container Day](https://youtu.be/qxWJRUF6JJc)
 - 05/14/2021 [Groupless Autoscaling with Karpenter @ Kubecon](https://www.youtube.com/watch?v=43g8uPohTgc)
 - 05/04/2021 [Karpenter @ Container Day](https://youtu.be/MZ-4HzOC_ac?t=7137)
+
+
+# Fix for Issue
```

Comment:
I see you have a few proposals for how we can fix this -- do you want to implement one of them (or extend our existing scripting logic) so that we fix this problem that we have right now?